src/main/java/nl/hsac/fitnesse/util/RandomDomain.java

```diff
@@ -3,11 +3,37 @@
 public class RandomDomain {
 
     private static final RandomUtil RANDOM_UTIL = new RandomUtil();
-    private static final String PERMITTED = "abcdefghijklmnopqrstuvwxyz1234567890-";
+    private static final String PERMITTED_NO_PERIOD = "abcdefghijklmnopqrstuvwxyz1234567890-";
+    private static final String PERMITTED_PERIOD = "abcdefghijklmnopqrstuvwxyz1234567890-.";
 
 
-    public String getRandomDomain(int length) {
-        return RANDOM_UTIL.randomString(PERMITTED, length);
+    public String getRandomSecondLevelDomain(int length) {
```

Comment:
Eh, never considered that.
Your solution:
- would work
- is simpler
- is probably more readable.

Ok, I will refactor. 😄